third-party/linshare/src/main/java/org/apache/james/linshare/client/GrantUserPermissionRequest.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.james.linshare.client;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import java.util.List;
+import java.util.Objects;
+
+public class GrantUserPermissionRequest {
```

Comment:
GrantUserPermissionRequest, UserCreationRequest, UserResult are only used in tests, maybe we can move them to the test package?

Reply:
GrantTechnicalAccountPermissionRequest